components/Icon/Icon.jsx

```diff
@@ -116,7 +117,9 @@ const Icon = ({ name, skin, size, ...props }) => {
 
   if (!components[name]) return <span>{name}</span>;
 
-  const SelectedIcon = components[name];
+  const SelectedIcon = styled(components[name])`
+    width: 24px;
```

Comment:
```suggestion
    max-width: 24px;
```
Faz sentido fazer assim? Se sim, acho que a intenção pode ficar mais clara

Reply:
@jgpacker, mas ai abre brecha para ter um icone com o tamanho menor, o que acredito que essa não seja a intenção. Correto @rapahaeru?

Reply:
Pelo que entendi, o problema são ícones gigantes.

Reply:
I changed to initial width to zero because when I tested with different colors and sizes it was behaving oddly, starting with black than changing the color. Now the icon will appear only when the final style is loaded.